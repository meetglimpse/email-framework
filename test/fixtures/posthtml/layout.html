---
greeting: Hello
---

<extends src="test/stubs/layouts/basic.html">
  <block name="template">
    Environment: {{ page.env }}

    Front matter variable: {{ page.greeting }}
  </block>
</extends>
