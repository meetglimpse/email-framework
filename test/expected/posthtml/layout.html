    Environment: maizzle-ci

    Front matter variable: Hello
